javascript/spread-operator.md

```diff
@@ -0,0 +1,34 @@
+### The Spread Operator
+
+A powerful feature of ES6, the latest standard of JavaScript implementation, is the spread operator, represented by three dots `...`.
+
+This operator is commonly used as a prefix for parameter names or elements in an array. In this way, expressions can be expanded in places where multiple arguments (for function calls) or multiple elements (for array literal notation) are expected.
+
+* *Example 1 - Function calls*   
+The spread syntax enables you to pull a real array out of the function arguments by adding a parameter prefixed by `...`. This is quite handy when an array is passed as an argument (or multiple arguments) to a function, as below:
```

Comment:
Hm, I'm confused what you mean by 'pull a real array out of the function arguments'? I'd say the role of the spread operator is the opposite - pull the function arguments our of an array?